### ISSUE TITLE
Add `paddle.nn.functional.norm` NN Norm Functions to PaddlePaddle Frontend

### ISSUE THREAD

**Issue by hmahmood24** [unifyai/ivy#15054]
Add [NN Norm Functions](https://www.paddlepaddle.org.cn/documentation/docs/en/api/index_en.html) to PaddlePaddle Frontend

> Please keep in mind that the proper way to link an issue to this list is to comment "- [ ] #issue_number" while the issue's title only includes the name of the function you've chosen.

- [ ] batch_norm
- [ ] instance_norm
- [ ] layer_norm
- [ ] local_response_norm
- [ ] normalize

**Comment by melch-inno** [comment]
batch_norm #17377

**Comment by Codehackerone** [comment]
Hello. I would like to work on this as the part for the assignment

**Comment by Madjid-CH** [comment]
- [ ] #19241

**Comment by ToniAnton22** [comment]
#19241 can I contribute to this issue?

**Comment by Madjid-CH** [comment]
> #19241 can I contribute to this issue?

It's already taken by me 😊
but you can help if you want.

**Comment by Madjid-CH** [comment]
#19696

**Comment by shaby112** [comment]
- [ ] #19696 
Can I work on this?

**Comment by IANXHULU** [comment]
Can contribute to this task

**Comment by Ambro19** [comment]
#22050

**Comment by Ambro19** [comment]
#22053

**Comment by Ambro19** [comment]
#22055

**Comment by Madjid-CH** [comment]
- [ ] #19696

**Comment by Ambro19** [comment]
Hey @hmahmood24 @Madjid-CH, @unify, I implemented instance_norm function (#22053) and its testing, but I am facing issue trying to do "pull request". When "I git push", I received this error: 
 **! [remote rejected]     adding-instance_norm-and-testing -> adding-instance_norm-and-testing (refusing to allow a Personal Access Token to create or update workflow `.github/workflows/array-api-det-coverage.yml` without `workflow` scope)
error: failed to push some refs to 'https://github.com/Ambro19/ivy.git'**
(ivy_dev) PS E:\Unify-Ivy\ivy>
  
And this is the coverge.yml file is: This is the issue's line: SSH_DEPLOY_KEY: ${{ secrets.SSH_DEPLOY_KEY } 
name: array-api-determine-test-coverage
on:
  workflow_dispatch:
permissions:
  actions: read
  
jobs:
  determine_coverage:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout Ivy 🛎
        uses: actions/checkout@v2
        with:
          path: ivy
          persist-credentials: false
          submodules: "recursive"
          fetch-depth: 150

      - name: Determine Test Coverage
        run: |
          pip install pydriller tqdm
          cd ivy
          python run_tests_CLI/array_api_det_coverage.py
          cd ..
          mkdir tests
          cp ivy/tests.pbz2 tests/

      - name: Push Mapping
        uses: cpina/github-action-push-to-another-repository@main
        env:
          **SSH_DEPLOY_KEY: ${{ secrets.SSH_DEPLOY_KEY }}**
        with:
          source-directory: tests/
          destination-github-username: 'unifyai'
          destination-repository-name: 'Mapping'
          user-email: rashul.chutani@gmail.com
          commit-message: Update Array API Tests Mapping
          target-branch: main

Any idea on how to solve this issue? Any will be appreciated.

**Comment by Madjid-CH** [comment]
> Hey @hmahmood24 @Madjid-CH, @unify, I implemented instance_norm function (#22053) and its testing, but I am facing issue trying to do "pull request". Any idea on how to solve this issue? Any help will be appreciated.
> 
> When "I git push", I received this error: **! [remote rejected] adding-instance_norm-and-testing -> adding-instance_norm-and-testing (refusing to allow a Personal Access Token to create or update workflow `.github/workflows/array-api-det-coverage.yml` without `workflow` scope) error: failed to push some refs to 'https://github.com/Ambro19/ivy.git'** (ivy_dev) PS E:\Unify-Ivy\ivy>
> 
> And below is the array-api-det-coverge.yml file. This is the issue's line: SSH_DEPLOY_KEY: ${{ secrets.SSH_DEPLOY_KEY }
> 
> name: array-api-determine-test-coverage on: workflow_dispatch: permissions: actions: read
> 
> jobs: determine_coverage: runs-on: ubuntu-latest steps: - name: Checkout Ivy 🛎 uses: actions/checkout@v2 with: path: ivy persist-credentials: false submodules: "recursive" fetch-depth: 150
> 
> ```
>   - name: Determine Test Coverage
>     run: |
>       pip install pydriller tqdm
>       cd ivy
>       python run_tests_CLI/array_api_det_coverage.py
>       cd ..
>       mkdir tests
>       cp ivy/tests.pbz2 tests/
> 
>   - name: Push Mapping
>     uses: cpina/github-action-push-to-another-repository@main
>     env:
>       **SSH_DEPLOY_KEY: ${{ secrets.SSH_DEPLOY_KEY }}**
>     with:
>       source-directory: tests/
>       destination-github-username: 'unifyai'
>       destination-repository-name: 'Mapping'
>       user-email: rashul.chutani@gmail.com
>       commit-message: Update Array API Tests Mapping
>       target-branch: main
> ```

Hi @Ambro19 
please refer to these PR : https://github.com/unifyai/ivy/pull/19261

**Comment by rajarshighoshal** [comment]
Hi I want to work on this issue if no one is working on it

**Comment by msarmadsohail** [comment]
- [ ] #22050

**Comment by Boghdady9** [comment]
#23300

**Comment by SamruddhiNavale** [comment]
- [ ] #24289

**Comment by parteekcoder** [comment]
- [] #15054

**Comment by Halloumopitta** [comment]
#23300 Hello.I am interested in this issue

**Comment by Halloumopitta** [comment]
Hello.I am interested in this issue #23300

**Comment by Halloumopitta** [comment]
- [] #23300

**Comment by abdullateefogundipe** [comment]
[27199](https://github.com/unifyai/ivy/issues/27199)
